CSCore/XAudio2/X3DAudio/X3DAudioCore.cs

```diff
@@ -47,6 +47,10 @@ private void LoadX3DAudio()
             _hModule = Win32.NativeMethods.LoadLibrary("X3DAudio1_7.dll");
             if (_hModule == IntPtr.Zero)
                 _hModule = Win32.NativeMethods.LoadLibrary("X3DAudio2_8.dll");
+            if (_hModule == IntPtr.Zero)
+                _hModule = Win32.NativeMethods.LoadLibrary("XAudio2_8.dll");
+            if (_hModule == IntPtr.Zero)
+                _hModule = Win32.NativeMethods.LoadLibrary("XAudio2_9.dll");
```

Comment:
Since Windows 10 always comes with XAudio2.8 and XAudio2.9 this won't be necessary. Also, XAudio2.9 is currently not supported by cscore.